loris/Dockerfile

```diff
@@ -3,16 +3,18 @@ FROM ubuntu:14.04
 RUN apt-get update
 
 RUN apt-get install -y python python-pip python-setuptools python-dev \
-    uwsgi-plugin-python
+    uwsgi uwsgi-plugin-python
```

Comment:
Why do we know include `uwsgi` by itself?

Reply:
We already did – this just puts our `apt-get install` commands on one line. Previously we were installing it after we’d installed Loris.

Reply:
Ah, I see it.